source/documentation/monitoring_apps/logs.md

```diff
@@ -0,0 +1,163 @@
+## Logs
+
+Cloud Foundry and apps running on Cloud Foundry can generate logs. If your app is failing to deploy, or crashing, and it's not clear why from the command line client messages, you should consult the logs.
+
+For log messages to be captured by Cloud Foundry, your application should be writing them to `STDOUT`/`STDERR`, rather than a log file.
+
+The most direct way to view events related to your application through the deploy process is:
+
+```bash
+cf logs APPNAME
+```
+
+Used alone, `cf logs` will tail the combined stream of logs from each Cloud Foundry service involved in your application deploy. Running with the `--recent` flag will stream the entire [Loggregator](https://docs.cloudfoundry.org/loggregator/architecture.html) [external link] buffer for your app.
```

Comment:
I think "stream" is maybe confusing when talking about the `cf logs --recent`. I'd expect streaming to mean the kind of continuous pushing of logs that you see when running `cf logs`. I think the behavior of `cf logs --recent` could be better described as "a dump of recent logs".